index.js

```diff
@@ -43,7 +54,10 @@ module.exports = exports = (app) => (args) => {
             req.query(args.__ow_query);
 
         if (args.__ow_body && (args.__ow_method === 'post' || args.__ow_method === 'put' || args.__ow_method === 'patch')) {
-            req = req.send(args.__ow_body);
+            var body = args.__ow_body
+            if (isBase64(body))
```

Comment:
For me, this worked for  content type `application/json` and with binary data in `multipart/form-data`